### PR TITLE
Fix Yarn Scripts and Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by gitaalekhyapaul**
**Issue** #6690

*Description of changes:*
- Updated the CONTRIBUTING.md to add `lerna` and `rimraf` and pre-requisite to setting up the dev environment. 
- Added escaped double quotes to the scripts in order to add support for pathnames with spaces in between them.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by princiya** [comment]
Thanks for improving the docs @gitaalekhyapaul 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This section should get you running with **Amplify CLI** and get you familiar wi
 1. Ensure you have [Node.js](https://nodejs.org/en/download/) installed, which comes bundled with [`npm`](https://github.com/npm/cli). Use it to install or upgrade [`yarn`](https://classic.yarnpkg.com/en/docs/install):
 
    ```sh
-   npm install --global yarn
+   npm install --global yarn lerna rimraf
```
Comment (edwardfoyle) [inline CONTRIBUTING.md:35]: I think we should add lerna and rimraf as devDependencies rather than expect them to be global

Comment (gitaalekhyapaul) [reply CONTRIBUTING.md:35]: Yes, I can do that. Although then I have to execute the `lerna` and `rimraf` commands with `npx`. Do you want me to do that?

Comment (gitaalekhyapaul) [reply CONTRIBUTING.md]: @edwardfoyle I have removed the global deps.

**Review by edwardfoyle — APPROVED** [review]
LGTM! Thanks for the contribution!

**Pull request merged**